components/landing/TheQuickStart/StartLocally.vue

```diff
@@ -76,96 +73,91 @@ type ChoicesGroup = {
-      [this.OPERATING_SYSTEMS.windows]: 'PrerequisitesForWindows'
-    }
+]
+
+const selectedOptions = reactive({
+  'qiskit-install': QISKIT_INSTALL.stable,
+  os: OPERATING_SYSTEMS.mac
+})
+
+const segmentLabel = 'Qiskit Install'
+
+const codeToInstallStableOnLinux = 'pip install -U pip && pip install qiskit'
+const codeToInstallStableOnMac = 'pip install qiskit'
+const codeToInstallStableOnWindows = 'pip install qiskit'
+
+const codeToInstallMasterOnLinux = 'pip install git+https://github.com/Qiskit/qiskit-terra git+https://github.com/Qiskit/qiskit-aer git+https://github.com/Qiskit/qiskit-ignis git+https://github.com/Qiskit/qiskit-aqua git+https://github.com/Qiskit/qiskit-ibmq-provider'
+const codeToInstallMasterOnMac = 'pip install git+https://github.com/Qiskit/qiskit-terra git+https://github.com/Qiskit/qiskit-aer git+https://github.com/Qiskit/qiskit-ignis git+https://github.com/Qiskit/qiskit-aqua git+https://github.com/Qiskit/qiskit-ibmq-provider'
+const codeToInstallMasterOnWindows = 'pip install git+https://github.com/Qiskit/qiskit-terra git+https://github.com/Qiskit/qiskit-aer git+https://github.com/Qiskit/qiskit-ignis git+https://github.com/Qiskit/qiskit-aqua git+https://github.com/Qiskit/qiskit-ibmq-provider'
+
+const prerequisites = {
+  [QISKIT_INSTALL.stable]: {
+    [OPERATING_SYSTEMS.linux]: null,
+    [OPERATING_SYSTEMS.mac]: null,
+    [OPERATING_SYSTEMS.windows]: null
+  },
+  [QISKIT_INSTALL.master]: {
+    [OPERATING_SYSTEMS.linux]: 'PrerequisitesForLinux',
+    [OPERATING_SYSTEMS.mac]: 'PrerequisitesForMac',
+    [OPERATING_SYSTEMS.windows]: 'PrerequisitesForWindows'
   }
+}
 
-  codeToInstall = {
-    [this.QISKIT_INSTALL.stable]: {
-      [this.OPERATING_SYSTEMS.linux]: this.codeToInstallStableOnLinux,
-      [this.OPERATING_SYSTEMS.mac]: this.codeToInstallStableOnMac,
-      [this.OPERATING_SYSTEMS.windows]: this.codeToInstallStableOnWindows
-    },
-    [this.QISKIT_INSTALL.master]: {
-      [this.OPERATING_SYSTEMS.linux]: this.codeToInstallMasterOnLinux,
-      [this.OPERATING_SYSTEMS.mac]: this.codeToInstallMasterOnMac,
-      [this.OPERATING_SYSTEMS.windows]: this.codeToInstallMasterOnWindows
-    }
+const codeToInstall = {
+  [QISKIT_INSTALL.stable]: {
+    [OPERATING_SYSTEMS.linux]: codeToInstallStableOnLinux,
+    [OPERATING_SYSTEMS.mac]: codeToInstallStableOnMac,
+    [OPERATING_SYSTEMS.windows]: codeToInstallStableOnWindows
+  },
+  [QISKIT_INSTALL.master]: {
+    [OPERATING_SYSTEMS.linux]: codeToInstallMasterOnLinux,
+    [OPERATING_SYSTEMS.mac]: codeToInstallMasterOnMac,
+    [OPERATING_SYSTEMS.windows]: codeToInstallMasterOnWindows
   }
+}
 
-  get selectedOs () : string {
-    return this.selectedOptions.os
-  }
+const selectedOs = computed<string>(() => selectedOptions.os)
 
-  getPrerequisitesToInstallQiskit () : string | null {
-    const { 'qiskit-install': qiskitInstall, os } = this.selectedOptions
+const getPrerequisitesToInstallQiskit = computed<string | null>(() => {
```

Comment:
This has changed from a function to a computed property, so I think we should rename it to follow our syntax:

Suggestion:
```ts
const prerequisitesToInstallQiskit = computed<string | null>(() => {
```

Reply:
Changed in https://github.com/Qiskit/qiskit.org/pull/2935/commits/6c29132b6f803247e856622cfa43b48e3562829d.